Example Apps/Example Swift/MenuManager.swift

```diff
@@ -11,6 +11,9 @@ import SmartDeviceLink
 import SmartDeviceLinkSwift
 
 class MenuManager: NSObject {
+
+    static var currentTemplate : SDLPredefinedLayout = .nonMedia
```

Comment:
I wonder if it might be better to pass a `refreshTemplateHandler` that returns the selected template to the `MenuManager` similar to how we pass a `refreshUIHandler` to the `ButtonManager`. That way we can remove setting the template in the menu manager and consolidate the code for setting the template in the `ProxyManager`.

Then the handler is triggered when the `currentTemplate` is set:
```
public fileprivate(set) var currentTemplate: SDLPredefinedLayout {
    didSet {
        guard let refreshTemplateHandler = refreshTemplateHandler else { return }
        refreshTemplateHandler(currentTemplate)
    }
}
```